quadrat/experimental-theme.json

```diff
@@ -202,8 +202,8 @@
 			},
 			"post-content": {
 				"padding": {
-					"left": "var(--wp--custom--margin--horizontal)",
-					"right": "var(--wp--custom--margin--horizontal)"
+					"left": "0px",
```

Comment:
I understand why you want to remove this, so that content can extend the full width. But it causes all of the blocks (including text) to start at the edge of the screen:

<img width="410" alt="Screen Shot 2021-05-20 at 11 52 38 AM" src="https://user-images.githubusercontent.com/5375500/119012274-bd533f00-b94a-11eb-9a7f-ebc84b4397f4.png">